src/UnitTests/LanguageServerClient/UnitTests.LanguageServerClient/PythonLanguageClient.cs

```diff
@@ -0,0 +1,273 @@
+﻿// Python Tools for Visual Studio
+// Copyright(c) Microsoft Corporation
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the License); you may not use
+// this file except in compliance with the License. You may obtain a copy of the
+// License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// THIS CODE IS PROVIDED ON AN  *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS
+// OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY
+// IMPLIED WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+// MERCHANTABILITY OR NON-INFRINGEMENT.
+//
+// See the Apache Version 2.0 License for specific language governing
+// permissions and limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Python.Core.Disposables;
+using Microsoft.Python.Parsing;
+using Microsoft.VisualStudio.LanguageServer.Client;
+using Microsoft.VisualStudio.Threading;
+using StreamJsonRpc;
+using LSP = Microsoft.VisualStudio.LanguageServer.Protocol;
+using Task = System.Threading.Tasks.Task;
+
+namespace UnitTests.LanguageServerClient {
+    /// <summary>
+    /// Implementation of the language server client.
+    /// </summary>
+    /// <remarks>
+    /// See documentation at https://docs.microsoft.com/en-us/visualstudio/extensibility/adding-an-lsp-extension?view=vs-2019
+    /// </remarks>
+    public class PythonLanguageClient : ILanguageClient, ILanguageClientCustomMessage2, IDisposable {
+        private readonly JoinableTaskContext _joinableTaskContext;
```

Comment:
this doesn't run in Vs and run in console, so does JoinableTaskContext actually get used? who set Joinable factory? if  no one is setting it up, we probably can remove it? in VS, VS platform sets it up before everything else I believe.

Reply:
this runs outside VS. so its not actually needed.  It leftover from previous copied PTVS stuff.